prac_07/myguitars.py

```diff
@@ -6,20 +6,68 @@
 """
 import csv
 from guitar import Guitar
-from operator import itemgetter
 
 
 def main():
-    guitars = []
+    guitars = []  # set guitars to the empty list
+    open_file(guitars)
+    name = input("New guitar name: ")
+    get_new_guitar(guitars, name)  # create Guitar object
+    guitars.sort()
+    print_guitars(guitars)
+    write_file(guitars)
+
+
+def write_file(guitars):
+    """Write list of files to csv"""
+    with open("guitars.csv", "w") as out_file:
+        writer = csv.writer(out_file)
+        for guitar in guitars:
+            writer.writerow(guitar)
+
+
+def print_guitars(guitars):
+    """Print every guitar in list"""
+    for guitar in guitars:
+        print(guitar)
```

Comment:
I'm not sure if this really needs to be a function, I just had this code in my main function. I don't think this is beneficial enough to be it's own separate function though